sdk/cosmos/azure_data_cosmos/Cargo.toml

```diff
@@ -37,6 +37,7 @@ workspace = true
 
 [features]
 default = ["hmac_rust"]
+_internal_c = [] # Internal feature used to expose additional types/functions as needed to build libcosmosclient.
```

Comment:
Might consider `cplusplus` like @LarryOsterman used for exposing some C bindings and binding-friendly types for `azure_core_amqp`.

Reply:
Don't want `cplusplus` because this is very explicitly C (though it'll work in C++ too), but I could do `c`. I actually started with that but wanted to make really clear that this feature basically only exists to be enabled by `cosmosclient`.

Reply:
Good point about not using "cplusplus" becuase it's C, but I still wouldn't preface it with "_" and mentioning "internal" also probably isn't needed. AMQP's `cplusplus` is also for "internal" use in the same manner. It's more like an API that is for "Microsoft use only". Maybe either "c" (awfully short, though) or "cbind" or something like that? Not too picky on the name. Even "internal_c" isn't so bad, but don't like the "_" prefix.

Reply:
If we wanted to unify for C/C++ bindings, maybe we could do that across the board e.g., `cbind` or something. /cc @LarryOsterman

Reply:
I've also seen `c_api` out there, but usually that's gating actual `extern "C"` APIs, which isn't quite what this does.